### ISSUE TITLE
Philips Wiz Smart Wi-Fi Motion Sensor

### ISSUE THREAD

**Issue by johanjongsma** [sbidy/wiz_light#46]
Hi,

I just bought a motion sensor. Do you think it can be supported by your integration?

Best, johan

**Comment by sbidy** [comment]
Currently not. So I work heavily to get the light integration into the HA core.
You can have a look into my pywizlight project. This is the base lib to communicate with the lights.
Feel free to fork it and try to integrated the sensor into the lib. After that it should be easy to integrate this into a HA integration.

So I'll close this issue. Maybe in the future will come back to this.

**Comment by simora** [comment]
I saw the Wiz motion sensor at Home Depot today. Wasn't sure if it would behave like the remote control (local wireless signal not actually 802.11 wifi). Is this motion sensor a direct Wifi participant? if so i'll pick one up and take a stab at it.

**Comment by chrisbl01** [comment]
It's like the remote, not 802.11, I took mine back to Home Depot.

**Comment by simora** [comment]
Thank you for the response. i won't be picking one up. will have to deal with my slow response zwave motion sensor. any suggestions for quality and responsive wifi motion sensors i can integrate into Home Assistant?

**Comment by johanjongsma** [comment]
@simora: I use mine and it works fine! Very responsive, especially considering the price. I believe the sensor connects to (one of the?) lights via Bluetooth. You can monitor the event of the light switching on to process in HASS. Not elegant, but it works.

Johan

**Comment by shaun-blake** [comment]
@johanjongsma, Could you give a little more detail on how you use the motion sensor with Home Assistant?  I setup up one wiz motion sensor connected to one wiz bulb in the wiz app. Then I configured the wiz bulb as a switch in Home Assistant. I created an automation that turns on 3 Hue bulbs when the wiz bulb turns on and the same for off.

The result is close.  When I toggle the switch in the Home Assistant UI, the hue bulbs follow immediately. However, when I trigger the wiz motion sensor, the wiz bulb comes on but it takes around a minute and a half before the hue bulbs are triggered. When the wiz bulb goes off after no motion is detected, the other bulbs follow right away.

Anybody know what the cause of the delay might be? The wiz bulb is version 1.21.0, so maybe it's related to #62?

**Comment by blkrx7** [comment]
I was hoping to use the remote/motion sensor as a nice out of the box trigger for HA.

I peeled the motion sensor apart and it uses a common ESP 8266 based chip. These have wifi, but I cannot find any bluetooth hardware. 

I captured un-associated 802.11 wifi packets from the Wiz remote, broadcasting to ff:ff..ff. Presumably the Wiz lights all listen on the air for these broadcasts and react accordingly. 

Guesswork: I don't think any HA operating in a VM can monitor raw wifi traffic, so these will never communicate with HA without an intermediary. However anyone operating HA direct on hardware that has the capability of monitoring raw wifi, may.

Has anyone poked at the serial interface on these chips? I wonder if these will connect to a specifically named SSID for things like firmware updates?
